tests/host_tools/uffd/Cargo.toml

```diff
@@ -5,13 +5,13 @@ authors = ["Amazon Firecracker team <firecracker-devel@amazon.com>"]
 edition = "2018"
```

Comment:
While we're at it, can we also update the edition to 2021 here (and same for the `demo_seccomp` crate in the security test directory)? We did it for the firecracker crates a few weeks ago